server/src/test/resources/csars/yaml/valid/lamp-csar/template.yaml

```diff
@@ -0,0 +1,97 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  inputs:
+    database_name:
+      type: string
+    database_user:
+      type: string
+    database_password:
+      type: string
+    database_port:
```

Comment:
This input is declared but never used. Anyways I'd drop this one -- my_mysql_port should be sufficient.